draft-ietf-core-coap-tcp-tls.md

```diff
@@ -687,128 +687,105 @@ option is understood but cannot be processed, the option documents the behavior.
 
 Capability and Settings messages are used for two purposes:
 
-* Capability options advertise a capability from the sender to the recipient.
+* Capability options advertise the capabilities of the sender to the recipient. 

```

Comment:
This *could* be read to mean that the set of capability options in one CSM advertises all capabilities.
Maybe we can stay singular: Each capability option advertises one capability...